test/poller.spec.js

```diff
@@ -1,4 +1,4 @@
-/* global it, describe, xit */
+/* global it, describe */
```

Comment:
**thought**: completely unrelated to the PR, this should be using an eslint `env` instead of globals